testsuite/tests/warnings/w32.ml

```diff
@@ -0,0 +1,7 @@
+(* from MPR#7624 *)
+
+let[@warning "-32"] f x = x
```

Comment:
It would be good to add tests with multiple bindings (`let ... and ...`), and also for the "unused rec" warning.

Reply:
I have added such tests; do you see any other combination worth testing?

Reply:
It's worth testing the case where warnings are disabled globally (e.g. for a sub-structure) and re-enabled explicitly for some bindings.

Reply:
Done.